hive-event-listeners/apiary-gluesync-listener/src/main/java/com/expediagroup/apiary/extensions/gluesync/listener/ApiaryGlueSync.java

```diff
@@ -377,8 +377,7 @@ private Date convertTableDate(final Integer lastAccessTime) {
       return null;
     }
     try {
-      final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
-      return dateFormat.parse(lastAccessTime.toString());
+      return new Date(lastAccessTime);
```

Comment:
I am not sure why we were formatting in this way. `LastAccessTime` is always an Integer and there is no possible way where we can parse an integer using SimpleDateFormat.

It looks like Hive tables usually lastAccessTime=0 so it returns null. But, Iceberg tables actually has a lastAccessTime value. So it goes into here.